tasks/main.yml

```diff
@@ -1,49 +1,93 @@
 ---
-# tasks file for slc_ansible_role_maildev
-- name: Check if we are in a flying circus VM
-  ansible.builtin.command: which fc-manage
-  register: is_flyingcircus
-  failed_when:
-    - is_flyingcircus.rc not in [0, 1]
-  changed_when:
-    - False
+- name: Flying Circus detection
+  block:
```

Comment:
Does it have any sense having one single task in a block?
Are you doing that for some kind of aesthetic reason?

Just wondering.